### PR TITLE
Edit Photo List View

### LINKED ISSUE #63: `Photo` component to render a new photo in photo list

### LINKED ISSUE #64: `PhotoListForm` checkbox inputs for each guest from `GuestList`

### LINKED ISSUE #68: User should be able to toggle guests to include in each photo

### PR DESCRIPTION
**Pull request by hannakim91**
# Pull Request

## What's this PR do? 

- [X] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Testing
- [ ] Styling
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Description of changes?
- Conditionals for view on wedding detail page to flow from guest list creation page to photo list creation page
- Form to add a new photo with description and guests to include

## What are the relevant tickets? 
* Addresses #65 
* Closes #63, Closes #64, Closes #68

## Questions/Next Steps:
- [ ] styling
- [ ] error handling/user restrictions for submit, etc
- [ ] async handling
- [ ] Fix types from `any` to proper ones where possible

### CODE REVIEW

File: src/components/Checkbox/Checkbox.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        type="checkbox" 
+        // name={`props ${i+1}`} 
+        value={props.name} 
+        // checked={props.isChecked}
+        onChange={() => props.toggleCheckMark(props.name)}
+      />
+  </div>
+  )
+}
+
```
Comment (ericberglund117) [inline src/components/Checkbox/Checkbox.tsx:23]: Thanks for building out this checkbox component!

File: src/components/Photo/Photo.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+const Photo: React.FC<Props> = ({
+  id,
+  photoNumber,
+  guests,
+  description,
+}) => {
+  return (
+    <article className="photoCard">
+      <h1 className="photoInfo">Photo {photoNumber}</h1>
+      <h2 className="photoInfo">Guests: {guests.length > 0 && guests.map((guest: any) => guest.name)}</h2>
```
Comment (ericberglund117) [inline src/components/Photo/Photo.tsx:18]: I like how for each photo, it only displays the guests' names who are in the photo. Well done!

File: src/components/PhotoListForm/PhotoListForm.tsx [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+    event.preventDefault()
+    const guestList = guestsOptions.filter((guest: any) => guest.isChecked)
+    console.log(guestList)
+    const newPhotoData = {
+      id: photoData.length + 1,
+      photoNumber: photoData.length + 1,
+      guests: guestList,
+      description: description
+    }
+    setPhotoData([...photoData, newPhotoData])
```
Comment (ericberglund117) [inline src/components/PhotoListForm/PhotoListForm.tsx:51]: I like the process of toggling the check, mapping the guests, setting them to state and then using that as the data for the filter upon submission

**Review by ericberglund117 — APPROVED** [review]

File: public/index.html [hunk 4]
```diff
@@ -11,12 +11,11 @@
       name="description"
       content="Web site created using create-react-app"
     />
-    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+
```
Comment (Kristiannmiller) [inline public/index.html:14]: Nice! This error was bugging me

File: src/apiCalls.tsx [hunk 5]
```diff
@@ -31,3 +31,17 @@ export const postAWedding = (wedding: any) => {
     alert('Could not add wedding')
   })
 }
+
+export const getWeddingGuests = () => {
```
Comment (Kristiannmiller) [inline src/apiCalls.tsx:35]: Thanks for building these out! Nice to know we're all set once this endpoint is ready

File: src/components/PhotoListForm/PhotoListForm.tsx (hunk 3 above)
Comment (Kristiannmiller) [reply src/components/PhotoListForm/PhotoListForm.tsx:51]: Me too! This is a really nice way to handle this, even if it was a little tricky

File: src/components/WeddingDetails/WeddingDetails.tsx [hunk 6]
```diff
@@ -27,44 +28,72 @@ const WeddingDetails: React.FC<IndividualWedding> = ({
...
-			} else if (view === "editListView") {
-				setDetailsView(false)
-				setPhotoListView(false)
-				setEditListView(true)
-			} else {
-				setDetailsView(true)
-				setPhotoListView(false)
-				setEditListView(false)
-			}
+	// probably should be GET for individualWedding and not just guests when BE is ready with that?
```
Comment (Kristiannmiller) [inline src/components/WeddingDetails/WeddingDetails.tsx:37]: Yeah, I agree that the GET should be for individualWedding

**Review by Kristiannmiller — APPROVED** [review]
This looks great! Thanks for getting this tricky component figured out!

**Pull request merged**